modules/ROOT/pages/import-api-specification-design-center.adoc

```diff
@@ -0,0 +1,44 @@
+= Importing an API Specification from Design Center
```

Comment:
```suggestion
= Import an API Specification from Design Center
```
So far, I have lost the Gerund Battle (-ing) and we are using imperative voice in titles....  GLOBAL